examples/cp4s/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "region" {
+  description = "The region name that the cluster is currently running in"
+}
+variable "resource_group_name" {
+  description = "The resource name that the cluster is currently running under"
+}
+variable "cluster_id" {
```

Comment:
I think we can pass even name of the cluster ... please check and update accordingly if required.